### PR TITLE
Bump 4.2.x to dev

### PR DESCRIPTION
**Pull request by giohappy**

### CODE REVIEW

File: geonode/__init__.py [hunk 1]
```diff
@@ -19,7 +19,7 @@
 
 import os
 
-__version__ = (4, 2, 4, "final", 0)
+__version__ = (4, 2, 4, "dev", 0)
```
Comment (etj) [inline geonode/__init__.py:22]: 4.2.5 dev

**Review by etj — CHANGES_REQUESTED** [review]
dev version is wrong

**Comment by giohappy** [comment]
> dev version is wrong

@etj what do you mean?
BTW I've bumped to 4.2.5

File: setup.cfg [hunk 2]
```diff
@@ -114,8 +114,8 @@ install_requires =
     pinax-ratings==4.0.0
 
     # GeoNode org maintained apps.
-    django-geonode-mapstore-client==4.2.0
-    geonode-importer==1.0.8
+    django-geonode-mapstore-client>=4.1.1,<5.0.0
```
Comment (etj) [inline setup.cfg:117]: are we sure about `>=4.1.1`, shouldnt it be `>=4.2.0` ?

**Review by etj — APPROVED** [review]

**Review by mattiagiupponi — APPROVED** [review]

**Pull request merged**